tests/test_user_role_service_v2.py

```diff
@@ -0,0 +1,305 @@
+import pytest
+import requests
+
+
+def set_custom_header_name(headers):
+    default_header = headers.get("NHSD-Session-URID")
+    return {"NHSD-URID": default_header}
+
+
+class TestUserRoles:
+    """A test suite for testing userrole in id tokens/headers/userinfo"""
+
+    happy_path_params = [

```

Comment:
I think these would be better as constants out of the class coz of the way they have to be accessed in parametrize. Just a style/preference thing, functionality is good

Reply:
da70d9960e5b8497073bb89c750283be070ecf71